docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md

```diff
@@ -417,7 +381,7 @@ INSERT INTO bank_transactions VALUES (100, 10002, 'EU', 'checking', 400.00, 'deb
 ERROR:  Illegal state: Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64: . Errors from tablet servers: [Illegal state (yb/client/transaction.cc:284): Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64]
 ```
 
-Because we've created a transaction table for us-west-2, YugabyteDB assumes by default that we want to run a transaction local to that region (using the transaction status table `system.transactions_us_west_2` we created in Step 1), but such a transaction cannot modify data outside of us-west-2.
+Because we have a tablespace with placement set to the us-west-2 region, YugabyteDB assumes by default that we want to run a transaction local to that region (using the transaction status table automatically created for the `us_west_2_tablespace` created in Step 1), but such a transaction cannot modify data outside of us-west-2.
```

Comment:
Too many "created" in short order :)

```suggestion
Because we have a tablespace with placement set to the us-west-2 region, YugabyteDB assumes by default that we want to run a transaction local to that region (using the automatically-generated transaction status table for the `us_west_2_tablespace` created in Step 1), but such a transaction cannot modify data outside of us-west-2.
```